pages/games.js

```diff
@@ -1,28 +1,35 @@
 import Layout from '../components/MyLayout.js'
-import MyImage1 from './img1'
-export default function About() {
+import Pic from '../components/Pic'
+import Des from '../components/Des'
+
+export default function Games() {
   return (
     <Layout>
-      <div>
-        <div className="title_bar">
-        <h1 className="title">Our Games</h1>
+      <div className="title_bar">
+        <h1 className="title">OUR GAME</h1>
+      </div>
+      <div className="content_bar">
```

Comment:
There's still code being repeated in the `content_bar` sections, we can put these in a component. But we can do this later